### ISSUE TITLE
feat: second press on cloud/skyline should refresh feed

### ISSUE THREAD

**Issue by julez1203** [mozzius/graysky#71]
#### Second press on cloud/skyline should refresh feed, like how it is done with press on notifications

**Comment by mozzius** [comment]
I did have this but removed it, because it was a little too easy to refresh by default. I'll try again and see if I can do better

**Comment by julez1203** [comment]
Yeah, there could be also this (blue) badge such as on twitter to load new posts. But the second press is a good solution in the mean time I think
